metamorph/processor.go

```diff
@@ -352,7 +351,7 @@ func (p *Processor) processExpiredTransactions() {
 					p.pm.RequestTransaction(item.Hash)
 				} else {
 					p.logger.Debug("Re-announcing expired tx", slog.String("hash", txID.String()))
-					p.pm.AnnounceTransaction(item.Hash, item.AnnouncedPeers)
+					p.pm.AnnounceTransaction(item.Hash, nil)
```

Comment:
Looks like we don't need second parameter anymore? 🤔

Reply:
Yes, the 2nd parameter is not needed here. But that function is defined in the p2p package: https://github.com/libsv/go-p2p/blob/d26b92c11b36216da12748833b580b41496c4dd5/PeerManager.go#L72